metals/src/main/scala/scala/meta/internal/pantsbuild/BloopPants.scala

```diff
@@ -620,7 +638,9 @@ private class BloopPants(
         path.filename != "bloop.settings.json" &&
         !generatedProjects(path.toNIO)
       }
-      .foreach { path => Files.deleteIfExists(path.toNIO) }
+      .foreach { path =>
```

Comment:
It looks like you're using an older version of Scalafmt, which is causing these diffs 🤔